File: Android/app/src/main/java/info/arthurribeiro/cloudplusplus/data/model/entity/FormSection.kt
package info.arthurribeiro.cloudplusplus.data.model.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.serialization.Serializable

@Entity(tableName = "form_sections")
@Serializable
data class FormSection(
    @PrimaryKey val id: String,
    val structureId: String,
    val index: Int,
    val fields: String,
)

File: iOS/CloudPlusPlus/Data/Model/FormSection.swift
//
//  FormSections.swift
//  CloudPlusPlus
//
//  Created by arthur gonzaga on 18/05/25.
//
import SwiftData

@Model
class FormSection {
    var id: String
    var structureId: String
    var index: Int
    var fields: String
    
    init(id: String, structureId: String, index: Int, fields: String) {
        self.id = id
        self.structureId = structureId
        self.index = index
        self.fields = fields
    }
}
